administrator/components/com_users/tmpl/methods/list.php

```diff
@@ -0,0 +1,141 @@
+<?php
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  com_users
+ *
+ * @copyright   (C) 2022 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+// Prevent direct access
+defined('_JEXEC') or die;
+
+use Joomla\CMS\Factory;
+use Joomla\CMS\HTML\HTMLHelper;
+use Joomla\CMS\Language\Text;
+use Joomla\CMS\Router\Route;
+use Joomla\CMS\Uri\Uri;
+use Joomla\Component\Users\Administrator\Model\MethodsModel;
+use Joomla\Component\Users\Administrator\View\Methods\HtmlView;
+
+// phpcs:ignoreFile
+/** @var HtmlView $this */
+
+HTMLHelper::_('bootstrap.tooltip', '.hasTooltip');
+
+/** @var MethodsModel $model */
+$model = $this->getModel();
+
+$this->document->getWebAssetManager()->useScript('com_users.two-factor-list');
+
+?>
+<div id="com-users-methods-list-container">
+	<?php foreach($this->methods as $methodName => $method):
+		$methodClass = 'com-users-methods-list-method-name-' . htmlentities($method['name'])
+			. ($this->defaultMethod == $methodName ? ' com-users-methods-list-method-default' : '');
+		?>
+		<div class="com-users-methods-list-method <?php echo $methodClass?> my-3 p-2 border border-1">
+			<div class="com-users-methods-list-method-header d-flex flex-wrap align-items-center gap-2">
+				<div class="com-users-methods-list-method-image pt-1 px-3 pb-2">
+					<img src="<?php echo Uri::root() . $method['image'] ?>"
+						 alt="<?php echo $this->escape($method['display']) ?>"
+						 class="img-fluid"
+					>
+				</div>
+				<div class="com-users-methods-list-method-title flex-grow-1 d-flex flex-column">
+					<h4 class="fs-4 p-0 m-0 d-flex gap-3 align-items-center">
+						<span class="me-1 flex-grow-1">
+							<?php echo $method['display'] ?>
+						</span>
+						<?php if ($this->defaultMethod == $methodName): ?>
+							<span id="com-users-methods-list-method-default-tag" class="badge bg-info me-1">
+								<?php echo Text::_('COM_USERS_MFA_LIST_DEFAULTTAG') ?>
+							</span>
+						<?php endif; ?>
+					</h4>
```

Comment:
```suggestion
					</h3>
```


The headings on the page were h1, h2, h4

It is important not to miss any therefore this is changed to h3